### PR TITLE
[Angular] Language is not preserved when navigating to another page

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
1. Create new jss app for selected framework (angular in this case): 'jss create angular-app angular --fetchWith REST -b release/18.0.0'
2. Create API KEY in the Sitecore instance
3. Run 'jss setup' to confogure app
4. Deploy configs to the Sitecore instance
5. Add da-DK language to the Sitecore instance
6. Deploy items for en and da-DK languages: 'jss deploy items -c -d', 'jss deploy items -c -d language da-DK'
7. Deploy app: jss deploy app -c -d
8. Run command: jss start:connected
9. Open page: http://localhost:3000/styleguide
10. Click 'Show in Danish' link
11. Click 'GraphQL' link in the header 

AR: 'GraphQL' page is opened in english language
ER: 'GraphQL' page is opened in danish language (empty page should be shown in our case since we don't have english version for it)

## Testing Details
- CSR
- SSR

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

### CODE REVIEW

File: samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class StyleguideSitecoreContextComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.contextSubscription = this.jssContext.state.subscribe((state) => {
-      console.log('The current Sitecore Context in styleguide-sitecore-context.component.ts is...', state);
+      // console.log('The current Sitecore Context in styleguide-sitecore-context.component.ts is...', state);
```
Comment (CobyPear) [inline samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts:26]: Do we want to remove this `console.log` completely?

File: samples/angular/src/app/routing/layout/layout.component.ts [hunk 2]
```diff
@@ -69,6 +69,6 @@ export class LayoutComponent implements OnInit, OnDestroy {
   onPlaceholderLoaded(placeholderName: string) {
     // you may optionally hook to the loaded event for a placeholder,
     // which can be useful for analytics and other DOM-based things that need to know when a placeholder's content is available.
-    console.log(`layout.component.ts: placeholder component fired loaded event for the ${placeholderName} placeholder`);
+    // console.log(`layout.component.ts: placeholder component fired loaded event for the ${placeholderName} placeholder`);
```
Comment (CobyPear) [inline samples/angular/src/app/routing/layout/layout.component.ts:72]: Same here, do we want this `console.log`?

**Review by CobyPear — CHANGES_REQUESTED** [review]
Looks good to me besides a few comments.

File: samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts (hunk 1 above)
Comment (ambrauer) [reply samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts:26]: Maybe leave these in but change to `console.debug`?

Comment (ambrauer) [reply samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts]: Re-introduced these as `console.debug`s

**Pull request merged**